ext/strings.go

```diff
@@ -1062,6 +934,49 @@ func makeMatcher(locale string) (language.Matcher, error) {
 	return language.NewMatcher(tags), nil
 }
 
+type callbackImpl struct {
+	args traits.Lister
+}
+
+func (c *callbackImpl) String(arg ref.Val, locale string) (string, error) {
+	return formatString(arg, locale)
```

Comment:
is there and advantage to calling out to a separate function here or can this be inlined?  (same for all the below functions)